### PR TITLE
Update Helsinki-design system from 0.13.3 to 0.17.0

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:
Updating helsinki-design-system to latest version (17.0)

Needed also fixing for accessibility test according to new icon accessibility rules: https://hds.hel.fi/components/icons#accessibility
- Mostly fixed by using `aria-hidden` for icon
- Few cases also where used `aria-labelledby`
- Also for external links (with icon) added SrOnly text telling that the link will be opened in new tab

### CODE REVIEW

File: src/domain/collection/collectionHero/CollectionHero.tsx [hunk 1]
```diff
@@ -40,7 +42,8 @@ const CollectionHero: React.FC<Props> = ({ collection }) => {
         {!!linkText && !!linkText && (
```
Comment (Sartonon) [inline src/domain/collection/collectionHero/CollectionHero.tsx:42]: seems like here is some old code that has !!linkText twice, could be removed now

**Review by Sartonon — COMMENTED** [review]
Looking pretty good :) Are couple added screen reader texts readable by screen reader?

File: src/common/components/dateSelector/DateSelector.tsx [hunk 2]
```diff
@@ -164,15 +165,26 @@ const DateSelector: FunctionComponent<DateSelectorProps> = ({
...
+              `commons.dateSelector.srOnly.${
+                isMenuOpen ? 'calendarOpened' : 'calendarClosed'
+              }`
+            )}
+          </SrOnly>
+          {isMenuOpen ? (
+            <IconAngleUp aria-hidden />
+          ) : (
+            <IconAngleDown aria-hidden />
+          )}
```
[the remark this answers is not in the document]
Comment (nenonja) [reply src/common/components/dateSelector/DateSelector.tsx:187]: Yep it seems so, the whole DataSelector component is not very accessible. When on input field, it doesnt tell you in what format you should input the date. Calendar is only accessible by arrow keys, but when moving around with arrows, voiceover remains silent. I can remove these changes and just use aria-hidden, but this component's accessibility should be fixed at some point.

File: src/common/components/multiSelectDropdown/MultiSelectDropdown.tsx [hunk 3]
```diff
@@ -253,13 +254,24 @@ const MultiSelectDropdown: React.FC<MultiselectDropdownProps> = ({
...
+              `multiSelectDropdown.srOnly.${
+                isMenuOpen ? 'dropdownOpened' : 'dropdownClosed'
+              }`
+            )}
+          </SrOnly>
+          {isMenuOpen ? (
+            <IconAngleUp aria-hidden />
+          ) : (
+            <IconAngleDown aria-hidden />
+          )}
```
[the remark this answers is not in the document]
Comment (nenonja) [reply src/common/components/multiSelectDropdown/MultiSelectDropdown.tsx:268]: Same here, I will remove it.

File: src/common/components/dateSelector/DateSelector.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/common/components/dateSelector/DateSelector.tsx:187]: yes there seems to be lot to improve

**Review by Sartonon — APPROVED** [review]
Great !

**Pull request merged**